scipy/stats/_continuous_distns.py

```diff
@@ -4171,6 +4171,121 @@ def _entropy(self, c):
 wrapcauchy = wrapcauchy_gen(a=0.0, b=2*pi, name='wrapcauchy')
 
 
+class gennorm_gen(rv_continuous):
+    """A generalized normal continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `gennorm` is [1]::
+
+                                     beta
+        gennorm.pdf(x, beta) =  ---------------  exp(|x|**beta)
+                                2 gamma(1/beta)
+
+    `gennorm` takes ``beta`` as a shape parameter.
+    For beta = 1, it is identical to a Laplace distribution.
+    For beta = 2, it is identical to a normal distribution.
```

Comment:
i know about the scaling issue. this is why it says it is identical to *a* normal distribution, not to *the* normal distribution ;)

some other distributions in scipy aren't normalized to unit variance with scale=1. hence, i saw no need in standardizing this one, and decided to stick with the more concise formulation.

will make the documentation more precise to say what choice of beta and scale corresponds to which Laplace and normal distributions.